.jenkins/pytorch/test.sh

```diff
@@ -34,6 +34,15 @@ if [[ "$BUILD_ENVIRONMENT" == *asan* ]]; then
     (cd test && ! python -c "import torch; torch._C._crash_if_aten_asan(3)")
 fi
 
+export ATEN_DISABLE_AVX=0
+export ATEN_DISABLE_AVX2=0
+if [[ "${JOB_BASE_NAME}" == *NO_AVX* ]]; then
```

Comment:
This will need both 'ATEN_DISABLE_AVX2=1' and 'ATEN_DISABLE_AVX=1'

Reply:
I actually have them controlled explicitly by the job name - one job will have the string `NO_AVX_NO_AVX2` and another will have `NO_AVX2`, so that it's easy to understand what the flags control